bin/templates/cordova/lib/prepare.js

```diff
@@ -25,7 +25,7 @@ const ManifestJsonParser = require('./ManifestJsonParser');
 const PackageJsonParser = require('./PackageJsonParser');
 const SettingJsonParser = require('./SettingJsonParser');
 
-module.exports.prepare = (cordovaProject, options) => {
+module.exports.prepare = function (cordovaProject, options) {
```

Comment:
Why use `function` keyword instead of arrow function syntax?

Reply:
It is needed for scoping purpose. The arrow function loses the proper scoping in this case and causes the platform to fail to be added.

Reply:
I got it.